site/content/_shared/variables-and-secrets.md

```diff
@@ -7,4 +7,11 @@ There are two ways to store configuration information in Checkly: Variables and
 - **Variables** are used to store non-sensitive information. Variables are shown in plaintext when being edited, on the check result page and in logs. Variables can be accessed via the CLI and API.
 - **Secrets** allow you to store sensitive data for use in checks. Once saved secrets are never shown in the UI or in logs. The secret value cannot be accessed via the CLI or API.
 
+{{< warning >}}
```

Comment:
I think this section should also be added to `/docs/groups/variables/#environment-variables-and-secrets`. Wdyt?